CMakeLists.txt

```diff
@@ -639,14 +610,14 @@ set(nuttx_libs ${nuttx_kernel_libs} ${nuttx_system_libs} ${nuttx_apps_libs}
 
 if(NOT CONFIG_ARCH_SIM)
 
-  # TODO: nostart/nodefault not applicable to nuttx toolchain
-  if(CONFIG_ARCH_TOOLCHAIN_TASKING)
```

Comment:
Why did you delete so many recent changes? Did you directly overwrite the current version with your private code????

Reply:
> Why did you delete so many recent changes? Did you directly overwrite the current version with your private code????

Sorry for my mistake here, I will fix soon